### ISSUE TITLE
Allow to customize excluded folders to not load specific config subfolders - or not loading JSON files at all

### ISSUE THREAD

**Issue by Vadorequest** [balderdashy/sails#2972]
I recently ran into an issue described here:
https://groups.google.com/forum/#!topic/sailsjs/-xQxjegEdd0

I think we need to have an option to *override* the default `options.excludeDirs` used in `sails-build-dictionnary` through the `.sailsrc` configuration file.

See https://github.com/balderdashy/sails-build-dictionary/blob/master/index.js#L68

I had to manually change this source code to override he options to ignore one of my folders that contains JSON file, otherwise it messed up the whole server at start.

Also, ignoring JSON file would be wise I believe, since all default Sails config file are JS files. And also because there is no point loading JSON files at all, since we can't set something equivalent to `module.export.blueprints` and so on.

I think the quickest fix is to not load JSON file using require-all. Is this possible to have this fixed in 0.11-x?

**Comment by zfrenchee** [comment]
+1

**Comment by miguelmota** [comment]
@zfrenchee +1

**Comment by sailsbot** [comment]
Thanks for posting, @Vadorequest.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
